backend/app/modules/uq/service/preliminary.py

```diff
@@ -60,3 +61,9 @@ def _get_gp_score(self, context: Context, address: str) -> float:
         if antisybil_status is None:
             return 0.0
         return antisybil_status[0]
+
+    def get_all(self, context: Context) -> List[Tuple[str, Decimal]]:
+        epoch_num = context.epoch_details.epoch_num
+
+        all_uqs = list_all_uqs_by_epoch(epoch_num)
+        return [(uq.user.address, uq.validated_score) for uq in all_uqs]
```

Comment:
i think we have N+1 issue here, so we may need to change the way of getting uqs:

```query = UniquenessQuotient.query.options(joinedload(UniquenessQuotient.user))```